workitem/link/link_repository.go

```diff
@@ -97,6 +93,22 @@ func (r *GormWorkItemLinkRepository) ValidateCorrectSourceAndTargetType(ctx cont
 	return nil
 }
 
+// CheckParentExists returns error if there is an attempt to create more than 1 parent of a workitem.
+func (r *GormWorkItemLinkRepository) CheckParentExists(ctx context.Context, targetID uint64, linkType *WorkItemLinkType) (bool, error) {
+	// TOOD(kwk): find a safer way to format an SQL query to get rid of this
```

Comment:
I've fixed it in 5ccb799. @DhritiShikhar make sure you fetch changes from your remote before pushing, since I've directly pushed to your remote.